Traceback (most recent call last):
  File "/Users/dorme/.pyenv/versions/3.7.7/lib/python3.7/site-packages/jupyter_cache/executors/utils.py", line 56, in single_nb_execution
    record_timing=False,
  File "/Users/dorme/.pyenv/versions/3.7.7/lib/python3.7/site-packages/nbclient/client.py", line 1087, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/Users/dorme/.pyenv/versions/3.7.7/lib/python3.7/site-packages/nbclient/util.py", line 74, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/Users/dorme/.pyenv/versions/3.7.7/lib/python3.7/site-packages/nbclient/util.py", line 53, in just_run
    return loop.run_until_complete(coro)
  File "/Users/dorme/.pyenv/versions/3.7.7/lib/python3.7/asyncio/base_events.py", line 587, in run_until_complete
    return future.result()
  File "/Users/dorme/.pyenv/versions/3.7.7/lib/python3.7/site-packages/nbclient/client.py", line 541, in async_execute
    cell, index, execution_count=self.code_cells_executed + 1
  File "/Users/dorme/.pyenv/versions/3.7.7/lib/python3.7/site-packages/nbclient/client.py", line 832, in async_execute_cell
    self._check_raise_for_error(cell, exec_reply)
  File "/Users/dorme/.pyenv/versions/3.7.7/lib/python3.7/site-packages/nbclient/client.py", line 740, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply['content'])
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
# Calculate correlograms for the residuals in the two models
simple.correlog <- with(data_sf, correlog(x, y, simple_resid, increment=distance, resamp=0))
sar.correlog <- with(data_sf, correlog(x, y, sar_resid, increment=distance, resamp=0))

# Convert those to make them easier to plot
simple.correlog <- data.frame(simple.correlog[1:3])
sar.correlog <- data.frame(sar.correlog[1:3])

# plot a correlogram for shorter distances
plot(correlation ~ mean.of.class, data=simple.correlog, type='o', subset=mean.of.class < 5000)
# add the data for the SAR model to compare them
lines(correlation ~ mean.of.class, data=sar.correlog, type='o', subset=mean.of.class < 5000, col='red')

# add a horizontal  zero correlation line
abline(h=0)
------------------

Error in dmat/increment: non-numeric argument to binary operator
Traceback:

1. with(data_sf, correlog(x, y, simple_resid, increment = distance, 
 .     resamp = 0))
2. with.default(data_sf, correlog(x, y, simple_resid, increment = distance, 
 .     resamp = 0))
3. eval(substitute(expr), data, enclos = parent.frame())
4. eval(substitute(expr), data, enclos = parent.frame())
5. correlog(x, y, simple_resid, increment = distance, resamp = 0)
ERROR: Error in dmat/increment: non-numeric argument to binary operator


